transformers/component_tests/queries/test_helpers/test_helpers.go

```diff
@@ -25,7 +25,7 @@ import (
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	shared2 "github.com/makerdao/vdb-mcd-transformers/transformers/shared"
```

Comment:
happy to replace `shared2` with a better name in places where we need both the separate repo and a mcd-version specific version from here, but also wondering if we might want to rename that package altogether and/or move functions to packages with more specific names

Reply:
In the interim you could use transformers_shared.